packages/traceloop-sdk/src/lib/tracing/ai-sdk-transformations.ts

```diff
@@ -381,14 +375,37 @@ export const transformAiSdkAttributes = (
   transformVendor(attributes);
 };
 
+const transformToolCalls = (span: ReadableSpan): void => {
+  if (
+    span.attributes["ai.toolCall.args"] &&
+    span.attributes["ai.toolCall.result"]
+  ) {
+    span.attributes[SpanAttributes.TRACELOOP_ENTITY_INPUT] =
+      span.attributes["ai.toolCall.args"];
+    delete span.attributes["ai.toolCall.args"];
+    span.attributes[SpanAttributes.TRACELOOP_ENTITY_OUTPUT] =
+      span.attributes["ai.toolCall.result"];
+    delete span.attributes["ai.toolCall.result"];
+  }
+};
+
 const shouldHandleSpan = (span: ReadableSpan): boolean => {
-  return span.name in HANDLED_SPAN_NAMES;
+  return span.instrumentationScope.name === "ai";
+};
+
+export const transformAiSdkSpanNames = (span: Span): void => {
+  if (span.name === TOOL_SPAN_NAME) {
+    span.updateName(`${span.attributes["ai.toolCall.name"] as string}.tool`);
+  }
+  if (span.name in HANDLED_SPAN_NAMES) {
+    span.updateName(HANDLED_SPAN_NAMES[span.name]);
```

Comment:
nice! this `updateName` is a new api i think so. if that's the case this change will break the otel-v1 tho if we merge it. 

just read about it, it exists since v1, makes me wonder why didn't we use it.